crates/biome_cli/tests/commands/lint.rs

```diff
@@ -107,6 +107,23 @@ fn ok_read_only() {
     assert!(result.is_ok(), "run_cli returned {result:?}");
 }
 
+#[test]
+fn ok_without_file_paths() {
+    let mut fs = MemoryFileSystem::default();
+    let mut console = BufferConsole::default();
+
+    let file_path = Path::new("check.js");
+    fs.insert(file_path.into(), FORMATTED.as_bytes());
+
+    let result: Result<(), biome_cli::CliDiagnostic> = run_cli(
+        DynRef::Borrowed(&mut fs),
+        &mut console,
+        Args::from([("lint"), ""].as_slice()),
+    );
+
+    assert!(result.is_ok(), "run_cli returned {result:?}");
```

Comment:
I think we forgot to add a snapshot assertion

Reply:
Ok, this is a test for ok (without lint errors ) case, I'll change this to a test that emits lint errors.